spec/classes/foreman_install_spec.rb

```diff
@@ -1,16 +1,9 @@
 require 'spec_helper'
 
 describe 'foreman::install' do
-  on_supported_os.each do |os, facts|
-    next if only_test_os() and not only_test_os.include?(os)
-    next if exclude_test_os() and exclude_test_os.include?(os)
-
+  on_os_under_test.each do |os, facts|
     context "on #{os}" do
-      let :facts do
-        facts.merge({
-          :concat_basedir => '/tmp',
```

Comment:
There are two more instances of concat_basedir in this file that could be removed.